css/style.css

```diff
@@ -6,3 +6,131 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+
+ /*full page background color */
+ html {
+    background-color:rgb(221, 240, 240)
+ }
+
+ /*header styling*/
+ .header-container {
+    
```

Comment:
Just make sure you get rid of the extra white space here

Reply:
Do you mean the empty space, line 17?

Reply:
Yeah 🙂